website/static/bluebook.csl

```diff
@@ -231,7 +241,7 @@
     <layout>
         <group>
             <names variable="author" suffix=", "/>
-            <text variable="title" font-style="italic" suffix=", Open Science Framework"/>
+            <text macro="title"/>
```

Comment:
Rather than defining a custom macro to determine the suffix (which amounts to `', {}'.format(publisher)`), why not just add another text block below sourced from the `publisher` variable? These two lines would end up being 
```
<text variable="title" font-style="italic" suffix=", "/>
<text variable="publisher" font-style="italic"/>
```

Reply:
neat. let me try that out.